packages/client/lib/files.js

```diff
@@ -81,7 +82,8 @@ class Files {
     if (opts.range) {
       opts.headers.Range = `bytes=${opts.range.from || 0}-${opts.range.to || ''}`
     }
-    return this.collection.fetch('/file/' + id, opts)
+    const query = opts.query || ''
```

Comment:
the fetch function being used here (client/lib/fetch.js) already supports a `params` option with an object to set query string in the URL. So lets not add anothet way here. Instead add a doc comment that passing `{ params: { dl: 1 }}` option will set the `Content-Disposition` header to `attachement` with the filename provided at upload, making the browser treat the file as a download (and not display in the browser).